spec/rails_helper.rb

```diff
@@ -51,4 +59,25 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  def login(email, password)
+    visit login_path
+    fill_in :email, with: email
+    fill_in :password, with: password
+    click_button "Login"
+  end
+# module Sorcery
+# module TestHelpers
+ # module Rails
+  #    def login_user_post(user, password)
+   #     page.driver.post(user_sessions_url, { email: user.email, password: password}) 
+    #  end
+   # end
+#  end
+ # end
+#  def login user, password = 'login'
+#    user.update_attributes password: password
+#
+ #   page.driver.post(user_sessions_url, {email: user.email, password: password})
```

Comment:
Line is too long. [81/80]<br>Incorrect indentation detected (column 1 instead of 2).